### DISCUSSION TITLE
Emote card to channel

### DISCUSSION THREAD

**Discussion by jeffboys123** [Chatterino/chatterino2#3164] [Ideas]
I recently discovered this feature in the ffz experiment tab and when you click on the emote in chat it gives you the option to sub to the channel, but I was thinking chatterino could have setting for this but just a link to the channel.
![image](https://user-images.githubusercontent.com/41973452/233733028-ef1a51ee-d832-4906-8c13-83d0e28ac3ca.png)
![MqRdZTM](https://user-images.githubusercontent.com/41973452/233733055-44e59777-3d48-42b9-be58-6c20c4329522.gif)

**Comment by apa420** [comment]
I might be wrong on this but I don't think that's possible right now.
Because of the service removal of TwitchEmotes.com there is no real way of determining where an emote came from.

https://discuss.dev.twitch.tv/t/get-emote-url-by-name-channel-by-emote-prefix-or-all-emotes-dump/33054/6

**Reply by Mm2PL** [reply]
If an emote is posted to chat, we get its ID which lets us query the API and in turn give us info about who owns the emote, etc.

**Reply by Mm2PL** [reply]
Here is a working mockup :tm: It's available here: https://github.com/Mm2PL/dankerino/tree/feature/emote_info_popup . `/xd` command brings up pajaDank emote info popup
![image](https://user-images.githubusercontent.com/41973452/233733648-2ebe9236-08e2-4927-a407-0c77f26bc224.png)

**Comment by jeffboys123** [comment]
or there could be something like this 🤔 

![image](https://user-images.githubusercontent.com/41973452/233732826-28373480-5b95-47b7-9d3c-8703f25ef4db.png)

**Reply by apa420** [reply]
#3114
